src/smoothscroll.js

```diff
@@ -0,0 +1,279 @@
+(function(WIN, DOC, undefined) {
```

Comment:
OK, so changing this to those names, though no idea about the server side rendering part.